cibernox/ember-power-select-typeahead
# Update to beta ember-power-select

Fixes #14:
- migrates to latest EPS API (
- Extend component rather than re-implementing it
- Add support for showing options on hover
- Simplify dirty detection on input value
- Add test for new option
- Add some demos to dummy app

I took some liberties with the code, but kept the old tests, which continue to pass. The only added functionality is the ability to pass `showOptions`, which causes the list to show on focus if `options` are provided (therefore providing a simple filtering typeahead instead of an async one). I added a test to cover this use case.

This likely needs a review to make sure I'm matching style properly, but I tested it pretty thoroughly (via the new views in the dummy app).

This also probably wants a minor/major version bump because of the breaking EPS dependency, but the API is identical so that's your call.

Here's a demo of the functionality:

![](http://oi.pxfx.io/1X1F1t2C2K45/Screen%20Recording%202016-07-28%20at%2004.20%20PM.gif)
